### DISCUSSION TITLE
blog/paper-semi-markov-modeling-for-cancer-insurance/

### DISCUSSION THREAD

**Discussion by giscus** [AntoineSoetewey/statsandr#97] [General]
# blog/paper-semi-markov-modeling-for-cancer-insurance/

Publication of my research paper entitled Semi-Markov modeling for cancer insurance in the European Actuarial Journal

https://statsandr.com/blog/paper-semi-markov-modeling-for-cancer-insurance/

**Comment by kimecha09** [comment]
Which table I can get the XR and XK  or do I have to do some calculations for that?
and for fig 2, what command do you use to make that fig?
thank you

**Reply by AntoineSoetewey** [reply]
Hello @kimecha09,

> Which table I can get the XR and XK or do I have to do some calculations for that?

Statbel's data can be found on their [website](https://statbel.fgov.be/en), and for XK/XR tables feel free to [contact me](https://statsandr.com/contact/) so I can send you the tables by email.

> and for fig 2, what command do you use to make that fig?

With the `plot()` function in R. It represents the mortality rates (from ***a***ctive to ***d***eath, hence $\mu^{ad}$) for the 3 different tables.

Hope this helps.

Regards,
Antoine

**Reply by kimecha09** [reply]
Thank you so much.
i'm sorry for asking this, but can I get the r coding of this paper ?

**Reply by AntoineSoetewey** [reply]
Please contact me via www.statsandr.com/contact and I will send you the code.

**Reply by kimecha09** [reply]
Oke Antoine, Thank you so much.
